mungegithub/mungers/approval-handler.go

```diff
@@ -103,11 +102,24 @@ func (h *ApprovalHandler) Munge(obj *github.MungeObject) {
 	}
 
 	approverSet := createApproverSet(comments)
+	//ownersMap: all necessary owners files -> people from files that have approved the PR
 	ownersMap := h.getApprovedOwners(files, approverSet)
 
+	ownersFiles := make([]string, len(ownersMap))
```

Comment:
Code is coming from:

https://github.com/kubernetes/test-infra/pull/1729/files#diff-019924d1fdee0dc57c792e7953adced4R328

It's needed in both places.

Though I can just pass as an arg the second time, but since it's derived from ownersMap which is also needed in the second instance, it might be okay to have it duplicated